src/media/playback/playback_device.cpp

```diff
@@ -380,7 +380,8 @@ void playback_device::resume()
             return;
 
         auto total_duration = m_reader->query_duration();
-        if (m_last_published_timestamp >= total_duration)
+        std::lock_guard< std::mutex > locker( m_last_published_timestamp_mutex );
```

Comment:
Do we want to wrap with {} to minimize the section?

Reply:
OK, I will wrap it